tests/test_object_creation.py

```diff
@@ -68,7 +68,14 @@ def test_rototrans_creation():
     np.testing.assert_array_equal(x=array, y=xr.DataArray(np.eye(4)[..., np.newaxis]))
     assert array.dims == dims
 
-    array = Rototrans(MARKERS_DATA.values, time=MARKERS_DATA.time)
+    data = Markers(MARKERS_DATA.values)
+    array = Rototrans.from_markers(
+        origin=data[:, [0], :],
+        axis_1=data[:, [0, 1], :],
+        axis_2=data[:, [0, 2], :],
```

Comment:
Is there any reason why you don't use `isel`?
I'm asking this because I've favored `isel` and `sel` in the code to be more explicit about the dimensions used, e.g.:
```python
    array = Rototrans.from_markers(
        origin=data.isel(channel=[0]),
        axis_1=data.isel(channel=[0, 1]),
        axis_2=data.isel(channel=[0, 2]),
        axes_name="xy",
        axis_to_recalculate="y",
    )
```

Reply:
Nope, did not know if one was preferable to the other.